ui/src/translations/en.json

```diff
@@ -84,5 +84,8 @@
   "no": "No",
   "volume_creation": "Volume creation",
   "volume_creation_success": "Volume {name} has been created successfully.",
-  "volume_creation_failed": "Volume {name} creation has failed."
+  "volume_creation_failed": "Volume {name} creation has failed.",
+  "volume_name_error": "Name must be lowercase letters, numbers and hyphens.",
+  "volume_path_error": "Device Path must start with a '/'",
+  "volume_size_error": "Size must be composed of a number and a unit like 250Gi or 1Ti."
```

Comment:
s/`number`/`positive integer`/ (and for the French s/`nombre`/`entier positif`/)